internal/cache/controller_obj_cache.go

```diff
@@ -2785,11 +2775,17 @@ func checkAndSetVRFFromNetwork(client *clients.AviClient) bool {
 		return true
 	}
 
-	networkName := lib.GetNetworkName()
-	if networkName == "" {
-		utils.AviLog.Warnf("Param networkName not specified, skipping fetching of the VRF setting from network")
+	networkList, err := lib.GetVipNetworkList()
+	if err != nil {
+		utils.AviLog.Warnf("Error gettting Network name: %s, skipping fetching of the VRF setting from network", err.Error())
+		return true
+	}
+
+	if len(networkList) == 0 {
```

Comment:
is it better to combine this with the if statement on line 2779?

Reply:
I think it is better to keep them separate. Error messages are different and it would be easier to debug.